sites/public/src/components/listings/search/ListingsSearchModal.tsx

```diff
@@ -145,9 +172,18 @@ export function ListingsSearchModal(props: ListingsSearchModalProps) {
           value={formValues.bathrooms}
         />
       </div>
-
       <div style={inputSectionStyle}>
         <div>Monthly Rent</div>
+        <input
+          type="text"
+          name="minRent"
+          value={formValues.minRent}
+          placeholder="Min Price: $"
+          style={textInputStyle}
+          onChange={(e: React.FormEvent<HTMLInputElement>) => {
+            updateValue("minRent", e.currentTarget.value)
+          }}
+        />
```

Comment:
There should be a hyphen in between these

Reply:
I'll add this in a future restyling PR.